### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dosumis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ To trigger an automated human readable diff, add the following tag to a comment
 
 CL is used in a number of applications including: 
 
+**CellxGene**
+
+The Cell ontology is used to annotate cell types on the single cell transcriptomics platform CellxGene.  All datasets on CellXGene are annotated according to a standard schema that specifies use of CL to record Cell Type.  Uses in CellXgene tools:
```
Comment (aleixpuigb) [inline README.md:62]:
```suggestion
The Cell ontology is used to annotate cell types on the single cell transcriptomics platform CellxGene.  All datasets on CellXGene are annotated according to a standard schema that specifies the use of CL to record Cell Type.  Uses in CellXgene tools:
```

File: README.md [hunk 2]
```diff
@@ -57,6 +57,14 @@ To trigger an automated human readable diff, add the following tag to a comment
...
+ - In the [Discover](https://cellxgene.cziscience.com/)) tool, the structure of CL is used to drive faceted searching for datasets and collections by Cell Type ,
+ - In the [Gene Expression](https://cellxgene.cziscience.com/gene-expression), CL annotation is used to aggregate expression data
+ - In [CellGuide](https://cellxgene.cziscience.com/cellguide) CL term names and synonyms drive search,  terms are used to structure the guide and ontology structure support a graphical browser
```
Comment (aleixpuigb) [inline README.md:65]: I am not sure I am undestanding this sentence correctly, but as I understand it:
 - In [CellGuide](https://cellxgene.cziscience.com/cellguide), CL term names and synonyms drive search  Terms are used to structure the guide, and the ontology structure supports a graphical browser.

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, consider the suggested changes.

**Comment by dosumis** [comment]
Fixed (Just realised I could have accepted your suggestions, but dit them by hand instead).

File: README.md [hunk 3]
```diff
@@ -57,6 +57,14 @@ To trigger an automated human readable diff, add the following tag to a comment
 
 CL is used in a number of applications including: 
 
+**CellxGene**
+
+The Cell ontology is used to annotate cell types on the single cell transcriptomics platform CellxGene.  All datasets on CellXGene are annotated according to a standard schema that specifies the use of CL to record Cell Type.  Uses in CellXgene tools:
+ - In the [Discover](https://cellxgene.cziscience.com/)) tool, the structure of CL is used to drive faceted searching for datasets and collections by Cell Type.
```
Comment (aleixpuigb) [inline README.md:63]:
```suggestion
 - In the [Discover](https://cellxgene.cziscience.com/) tool, the structure of CL is used to drive faceted searching for datasets and collections by Cell Type.
```

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
There is an extra closing parenthesis, but everything else looks good. Once you push the commit I will approve the PR.

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**